contrib/ci/check-cli-actions.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# pylint: disable=invalid-name,missing-module-docstring,missing-function-docstring
+#
+# Copyright 2025 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import subprocess
+import sys
+
+if __name__ == "__main__":
+    rc: int = 0
+
+    # load the manpage, bash-completion, etc
+    data: dict[str, str] = {}
+    for fn in [
+        "src/fwupdmgr.md",
+        "data/bash-completion/fwupdmgr",
+        "data/fish-completion/fwupdmgr.fish",
+    ]:
+        with open(fn, "rb") as f:
+            data[fn] = f.read().decode()
+
+    # if we can't run the binary for some reason, assume everything is okay
```

Comment:
what's the context you think this would happen?  We could always set `SKIP` if it's a CI only thing or something.